src/components/GameOfLife.js

```diff
@@ -0,0 +1,70 @@
+import React, {useState, useEffect} from 'react'
+
+const GameOfLife = () => {
+    const [board, setBoard] = useState(initialBoard())
+        
+    useEffect(() => {
+        setTimeout(() => {
+            setBoard(incrementTime(board))
+        }, 100)
+    }, [board])
+
+
+    return (
+    <div>{
+        board.map((row, rowIndex) => (
+            <><div key={rowIndex} style={{display: 'flex'}}>{
+            row.map((cell, cellIndex) => 
+                <div key={cellIndex} style={{backgroundColor: (cell ? 'white' : 'black'), height: '10px', width: '10px'}}></div>
+)}</div></>
+        ))
+        }</div>
+    )
+}
+
+export default GameOfLife
+
+const initialBoard = () => {
+    let board = Array(75)
```

Comment:
Magic number detected! :P I made the same mistake. :) Best to make it into a constant with a descriptive name.